docs/general/polkadot-vault.md

```diff
@@ -0,0 +1,135 @@
+|              Hardware designed to stay offline and be secure. Certified by French cybersecurity agency [ANSSI](https://www.ssi.gouv.fr/en/cybersecurity-in-france/the-national-cybersecurity-agency-of-france/).               |                                                                                                                                                                                                                                             Hardware not designed to stay offline. The user must switch off all inbound and outbound connections (network, wifi, Bluetooth, NFC).                                                                                                                                                                                                                                              |
+|                                 Hardware is resistant to side-channel attacks via [Secure Element](https://www.ledger.com/academy/security/the-secure-element-whistanding-security-attacks/).                                  |                                                                                                                                                                                                                                                                            If there is no wired connection, side-channel attacks are not possible.                                                                                                                                                                                                                                                                             |
+|                                                                              One mnemonic seed phrase for all intra- and inter-currency accounts.                                                                              | One mnemonic for each account or one mnemonic for multiple accounts via [account derivation](../learn/learn-account-advanced.md#derivation-paths) or default [Substrate address format](../learn/learn-account-advanced.md#address-format). Although the general recommendation is to use different accounts on different chains, the final choice depends on the user's needs. See [this support article](https://support.polkadot.network/support/solutions/articles/65000103707-can-i-use-the-same-account-on-polkadot-kusama-and-parachains-#Should-I-use-the-same-account-across-different-chains?) for more information. |
```

Comment:
Avoid the usage of terms like "currency".  Also, Ledger derives all the accounts through a single seed phrase right? If not correct the update I made. Also, pointing the user to the support article without making a general recommendation.


```suggestion
|                                                                              Accounts derived from one mnemonic seed phrase.                                                                              | One mnemonic for each account or one mnemonic for multiple accounts via [account derivation](../learn/learn-account-advanced.md#derivation-paths) or default [Substrate address format](../learn/learn-account-advanced.md#address-format). See [this support article](https://support.polkadot.network/support/solutions/articles/65000103707-can-i-use-the-same-account-on-polkadot-kusama-and-parachains-#Should-I-use-the-same-account-across-different-chains?) for information on whether to use the same account or different accounts on different chains based on your needs. |
```